launcher/launcher_unix.go

```diff
@@ -1,13 +1,20 @@
+//go:build !windows
```

Comment:
Does the `//go:build !windows` syntax supersede `//+build !windows` or do we need both?

Reply:
It looks like `go:build ...` is preferred to `+build ...` as of Golang 1.17. My guess is that our toolchain was automatically adding the new preferred tag syntax, but not removing the older syntax for backwards compatibility reasons.

I went ahead and removed the old syntax.